lib/colonel_kurtz/block_types.ex

```diff
@@ -48,17 +51,26 @@ defmodule ColonelKurtz.BlockTypes do
     |> to_block_type_struct
   end
 
-  @spec block_type_module(binary) :: {:ok, module} | {:error, atom}
+  @spec block_type_module(binary) :: module
```

Comment:
This is now guaranteed to return a module. It also emits warnings instead of errors.